tests/containers/registry.pm

```diff
@@ -62,8 +62,11 @@ sub run {
     $self->select_serial_terminal;
 
     # Install and check that it's running
-    add_suseconnect_product('PackageHub', undef, undef, undef, 300, 1) if is_sle(">=15");
-    my $pkg = is_tumbleweed ? 'distribution-registry' : 'docker-distribution-registry';
+    my $pkg = 'distribution-registry';
+    if (is_sle("<15-SP4")) {
+        add_suseconnect_product('PackageHub', undef, undef, undef, 300, 1) if is_sle(">=15");
+        $pkg = 'docker-distribution-registry';
```

Comment:
Do we run this on Leap as well? There it would need this line as well

Reply:
Oh yes, I forgot! I also wonder what's the plan for Leap 15.4 onwards.